docs/book/src/user/troubleshooting.md

```diff
@@ -161,7 +161,7 @@ clusterctl init --infrastructure docker
 ```
 ```bash
 Fetching providers
-Installing cert-manager Version="v1.8.2"
+Installing cert-manager Version="v1.9.1"
```

Comment:
Let's keep the old version in this doc. It's not super relevant but this error will only occur with old CAPI versions and thus never with cert-manager v1.9.1